src/server/pfs/server/driver.go

```diff
@@ -1878,7 +1961,7 @@ func (d *driver) flushCommit(pachClient *client.APIClient, fromCommits []*pfs.Co
 	return nil
 }
 
-func (d *driver) deleteCommit(txnCtx *txnenv.TransactionContext, userCommit *pfs.Commit) error {
+func (d *driver) deleteCommit(txnCtx *txnenv.TransactionContext, userCommit *pfs.Commit, force bool) error {
```

Comment:
not sure if we maybe want to keep this as an option for use elsewhere, or just get rid of it

Reply:
I'd get rid of it, my general feeling on `force` flags is that they shouldn't be needed if you have the right limitations on deletion. They can be a nice escape hatch though for when we have bugs, which happens often enough, that they're worthwhile sometimes. But it seems like you don't need it here so I'd just get rid of it until we do.